backend/btrixcloud/colls.py

```diff
@@ -156,20 +158,62 @@ async def find_collections(self, oid: uuid.UUID, names: List[str]):
         return [result["name"] for result in results]
 
     async def list_collections(
-        self, oid: uuid.UUID, page_size: int = DEFAULT_PAGE_SIZE, page: int = 1
+        self,
+        oid: uuid.UUID,
+        page_size: int = DEFAULT_PAGE_SIZE,
+        page: int = 1,
+        sort_by: str = None,
+        sort_direction: int = 1,
+        name: Optional[str] = None,
     ):
         """List all collections for org"""
+        # pylint: disable=too-many-locals
         # Zero-index page for query
         page = page - 1
         skip = page * page_size
 
         match_query = {"oid": oid}
 
-        total = await self.collections.count_documents(match_query)
+        if name:
+            match_query["name"] = name
+
+        aggregate = [{"$match": match_query}]
+
+        if sort_by:
+            if sort_by not in ("name", "description"):
```

Comment:
I wonder if sorting by description will be useful.. i guess possibly?
Perhaps should also add an index for oid + description.